ROMFS/px4fmu_common/init.d/airframes/4900_crazyflie

```diff
@@ -1,91 +0,0 @@
-#!/bin/sh
-#
-# @name Crazyflie 2
```

Comment:
This could probably be merged with the 4901_crazyflie21, but it might be better to leave it alone unless you want to review in detail.

What I actually wanted to do in special cases like the crazyflie is have an airframe that's part of the board support rather than carried centrally like we have now.

Reply:
Additionally special boards like the crazyflie could have ONLY their airframe and not even include the full selection in the build.

Reply:
ah we still support the build of the crazyflie 1? 
https://github.com/PX4/PX4-Autopilot/blob/f00d97d974dad5ef05df8a79c79afcb61daa3180/.ci/Jenkinsfile-compile#L51 seems to imply that.

Reply:
should I bring it back or can we even remove the support for the build target?